qipr_approver/approver/forms/project_form.py

```diff
@@ -83,30 +83,18 @@ def __init__(self, project=Project(),is_disabled=False):
                             'placeholder': 'Give a description of your Quality Improvement project here (Please use at least 250 words. When filled, this input box holds roughly 250 words.)',
                             'value': project.description or ''}
 
-        self.objective = {'name': 'objective',
-                          'type': 'text',
-                          'input_classes': ['textarea__height'],
-                          'placeholder': 'Give a brief description about your Quality Improvement project\'s objectives (up to 250 words)',
-                          'value': project.objective or ''}
-
-        self.scope = {'name': 'scope',
-                      'type': 'text',
-                      'input_classes': ['textarea__height'],
-                      'placeholder': 'Give a brief description about your Quality Improvement project\'s scope (up to 250 words)',
-                      'value': project.scope or ''}
+        self.overall_goal = {'name': 'overall_goal',
+                             'type': 'text',
+                             'input_classes': ['textarea__height'],
+                             'placeholder': 'Give a brief description about your Quality Improvement project\'s objectives (up to 250 words)',
```

Comment:
This placeholder is for the ex-objective field. Please update it for goal. 
"The overall goal of this project is: (Why are you doing this, what is it that's driving you?)"